+nosnoc/+discrete_time_problem/Cls.m

```diff
@@ -813,7 +813,7 @@ function generate_complementarity_constraints(obj)
                         G_ij = {};
                         H_ij = {};
                         sum_y_gap = obj.w.Y_gap(ii,jj) + sum2(obj.w.y_gap(ii,jj,:));
-                        for kk=1:(opts.n_s + rbp)
+                        for kk=1:opts.n_s
```

Comment:
why cls changes here? :D

Reply:
Because $\lambda_n$ calculations are not done at the rbp as they are free. This is the bugfix mentioned.